static/app/components/layouts/thirds.tsx

```diff
@@ -4,24 +4,36 @@ import NavTabs from 'sentry/components/navTabs';
 import space from 'sentry/styles/space';
 
 /**
- * Base container for 66/33 containers.
+ * Main container for a page.
  */
-export const Body = styled('div')<{noRowGap?: boolean}>`
-  padding: ${space(2)};
-  margin: 0;
-  background-color: ${p => p.theme.background};
-  flex-grow: 1;
+export const Page = styled('main')<{withPadding?: boolean}>`
+  display: flex;
+  flex-direction: column;
+  flex: 1;
+  ${p => p.withPadding && `padding: ${space(3)} ${space(4)}`};
```

Comment:
By default now does NOT have padding, use `withPadding` to add padding to the page content.